### DISCUSSION TITLE
トラックボールの上下のみ異常に反応する

### DISCUSSION THREAD

**Discussion by OTR7190** [Yowkees/keyball#417] [Q&A] [answered]
左右の移動においてはもともと使っていたトラックボールマウスと同じ動きなのですが、上下の動きが異常に反応しすぎてどこにカーソル移動したかわからなくなってしまう状態です。
半田付けの際の問題なのかと考えておりますが、確認ポイントありますでしょうか。
教えていただけると幸いです。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

ご連絡いただいた症状やカーソル移動が飛んだり震えたりという症状は、光学センサーの半田付けに不具合がある場合が多いです。
トラックボール基板の半田付けを見直していただき、半田が山型になっていない部分や隣とショートしているものがないかご確認ください。
また、L字コンスルーやその近くのジャンパー、ProMicro裏の半田付けも再度ご確認ください。

直らないようでしたら半田付け面のお写真をみせていただければ何かアドバイスできるかも知れません。

以上、お手数ですがご確認お願いいたします。

**Reply by OTR7190** [reply]
ご連絡ありがとうございます。
確認して改善しないようでしたら写真アップさせていただきます。
まずはお礼まで。

**Reply by OTR7190** [reply]
お世話になっております。
トラックボールが反応しなくなる事象が出始めてしまい悪化させてしまいました。
写真アップしてみますので、アドバイスいただけると幸いです。

![trashed-1702449320-20231113_153425](https://github.com/Yowkees/keyball/assets/149983121/eaf21236-a029-4539-b63c-b09582fbf889)
![trashed-1702449638-20231113_153143](https://github.com/Yowkees/keyball/assets/149983121/46015a08-ce81-42fc-b8cf-c83a4e68041c)

**Comment by Yowkees** [comment]
お写真ありがとうございます。

とてもキレイに半田付けできていると思いますので、コンスルーの接触不良でしょうか。
下写真のように、コンスルーピンが少しでも変形していると導通不良となることがありますので、ピンの形状をご確認ください。
また、トラックボール基板が奥まで挿入できているかもご確認ください。

以上、お手数ですがご確認お願いいたします。
![2_conthru_henkei](https://github.com/Yowkees/keyball/assets/84209772/b14e21ce-193e-44b4-a339-172038696e09)

**Reply by OTR7190** [reply, marked as answer]
ノートPC用のドッキングステーション経由で利用していると発生するという事が判明いたしました。
※いろいろバラして組み立て直して動作確認する時に、ノートPC直で差したら動きました

原因は特にわかってませんが、ノートPC直差しで利用を継続します。

ご協力ありがとうございました。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
USBケーブルの接続ですか！？ちょっと私も、そのような挙動になる原因が想像できませんが、しばらく様子を見ていただければ幸いです。ひとまず正常にもどり安心しました。
